packages/apollos-api/src/data/live/data-source.js

```diff
@@ -35,7 +39,7 @@ export default class LiveStream extends RESTDataSource {
     if (!channelId) return null;
 
     return {
-      isLive: this.getIsLive.bind(this, { channelId }),
+      isLive: await this.getIsLive.call(this, { channelId }),
```

Comment:
Second issue - we were expecting a value when calling this function, but we were originally returning a bound function that when called, would resolve to an object containing an unresolved promise.